pkg/collector/corechecks/system/cpu/cpu_windows.go

```diff
@@ -149,12 +148,12 @@ func (c *Check) Configure(integrationConfigDigest uint64, data integration.Data,
 	}
 
 	// do nothing
-	info, err := cpuInfo()
+	info := cpuInfo()
+	cpucount, err := info.CPULogicalProcessors.Value()
```

Comment:
`GetCpuInfo` never returned a non-`nil` error, in case of error this code would just use the default value, but as it really seemed like the intent was to return in case of error I kept it that way.

I can change that behavior if @DataDog/windows-agent wants to just ignore the error.